### ISSUE TITLE
How to get the form datas of each step

### ISSUE THREAD

**Issue by iamsajithsasi** [saini-g/react-step-progress#17]
I'm wondering, how the data of each form/steps needs to be obtained. Thanks

**Comment by iamsajithsasi** [comment]
function onFormSubmit() {
    //  what is the method to get the form datas ?
  }    

<StepProgressBar
        startingStep={0}
        onSubmit={onFormSubmit}
        submitBtnName="Add Company"
        steps={[
          {
            label: "1",
            name: "StepForm1",
            content: step1Content,
            validator: form1Data,
          },
          {
            label: "2",
            name: "StepForm1",
            content: step2Content,
          },
        ]}
      />

**Comment by saini-g** [comment]
@iamsajithsasi this feature is not yet implemented, would it be possible for you to raise a PR??

**Comment by nurinfazil** [comment]
@iamsajithsasi did you find a workaround? I'm having the same problem.

**Comment by iamsajithsasi** [comment]
> @iamsajithsasi did you find a workaround? I'm having the same problem.

Nope. Form were removed from dom during next/prev steps. I had to move owing to time constraints

**Comment by fernando-pineda** [comment]
I made this:

https://pastebin.com/J6AQUBEu

**Comment by amar-dot** [comment]
Actually i did tried to access data from each individual for and i am getting it so on submit i am pushing all the data at a time,i used parent child concept to pull up the data ehre from child i am passing the data to parent,but i stuck in problem like if i go back in the prev form the data just diappear but all the data are present in the variable but its not showing in the text field after moving forward and comming back any one have solution regarding this then plz help???
